pydss/flapper.py

```diff
@@ -33,16 +40,9 @@ def move(self, adr,amt):
 */
 """
 class Flapper:
-    wards = dict()
-    #messageclass
-    message = 0;
-
-    def rely(self, usr,send):
-        if self.wards[send] != 1:
-            self.wards[usr] = 1
-    def deny(self, usr,send):
-        if self.wards[send] != 1:
-            self.wards[usr] = 0
+
+
+
 #bid info
     class Bid :
         bid = 0;
```

Comment:
here as well, be sure to remove these static variables and instead instantiate them in an `__init__()` function, and add documentation